STM32/AC/Core/Src/ACBoard.c

```diff
@@ -52,23 +65,59 @@ static CAProtocolCtx caProto =
         .portState = CAportState,
 };
 
+/***************************************************************************************************
+** PRIVATE FUNCTIONS
+***************************************************************************************************/
+
+/*!
+** @brief Verbose print of the AC board status
+*/
+static void printAcStatus()
+{
+    static char buf[600] = { 0 };
+    int len = 0;
+
+    if (bsGetStatus() & AC_BOARD_VERSION_ERROR)
```

Comment:
Couldn't this be compressed to a call to the statusInfo?

Reply:
This was actually removed, but the push didn't happen.